core/modules/tiddler-change.js

```diff
@@ -0,0 +1,87 @@
+/*\
+title: $:/core/modules/tiddler-change.js
+type: application/javascript
+module-type: global
+
+Tiddler change handling utilities
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+function TiddlerChangeManager(options) {
+	var self = this;
+	options = options || "";
+	this.tiddlerChangeList = [],
+	this.tiddlerActionList = [];
+	this.updateTiddlerChangeLists(this.getTiddlerList());
+	$tw.wiki.addEventListener("change",function(changes) {
+		self.handleShortcutChanges(changes);
+	});
+}
+
+TiddlerChangeManager.prototype.getTiddlerList = function() {
+	return $tw.wiki.getTiddlersWithTag("$:/tags/TiddlerChangeAction");
+};
+
+TiddlerChangeManager.prototype.updateTiddlerChangeLists = function(tiddlerList) {
+	this.actionTiddlers = tiddlerList;
+	for(var i=0; i<tiddlerList.length; i++) {
+		var title = tiddlerList[i],
+			tiddlerFields = $tw.wiki.getTiddler(title).fields;
+		this.tiddlerChangeList[i] = tiddlerFields["change-tiddler"] !== undefined ? tiddlerFields["change-tiddler"] : undefined;
```

Comment:
I think, the field name should be different. IMO it contains the "triggering-tiddler" or may be "watch-tiddler" which I do like more.

Reply:
I like them both @pmario